auth0/src/test/java/com/auth0/android/request/internal/BaseRequestTest.kt

```diff
@@ -187,33 +200,28 @@ public class BaseRequestTest {
             resultAdapter,
             errorAdapter
         )
-        val networkError = mock<JsonIOException>()
+        val networkError = JsonIOException("Network error")
```

Comment:
why is a real instance prefered vs a mock?

Reply:
I think here for such a simple class using a mock might be excessive (but that is probably personal preference).
The advantage of using a real class here is that you can use a custom exception message which might make debugging test failures easier.